StringCalculator/StringCalculator.cs

```diff
@@ -0,0 +1,64 @@
+﻿namespace StringCalculator;
+public class StringCalculator
+{
+    private string[] seperators = new string[] { ",", "\\n" };
+    private List<string> negativeNumbers = new List<string>();
+    private int maxNumberToAdd = 1000;
+    private int finalResult = 0;
```

Comment:
The problem you can often run into with global variables is that they can hinder your readability. When reading code, if something is global and being manipulated in a method further down the class, it's not always easy to reason with that code because the variable seemingly comes out of nowhere.

Additionally, there is a concept called "cohesion", which refers to how well a global variable fits in a class: if the variable is used in most/all of the functionality of that class, it's said to have good cohesion. Variables with bad cohesion can also be a sign that your class is dealing with too many responsibilities.